static/src/javascripts/projects/commercial/modules/article-body-adverts.js

```diff
@@ -2,7 +2,8 @@
 import config from 'lib/config';
 import detect from 'lib/detect';
 import fastdom from 'lib/fastdom-promise';
-import abUtils from 'common/modules/experiments/utils';
+import * as abUtils from 'common/modules/experiments/utils';
```

Comment:
Since only one method in `abUtils` is being used in this module, can we just import that one method? It will make the code easier to grep, and it will allow Webpack to tree-shake.